content/03.methods.md

```diff
@@ -77,6 +79,20 @@ Recall that for each target gene and each -omics dataset, we ran 2 replicates of
 To make a statistical comparison between two models using these performance distributions, we used paired-sample _t_-tests, where performance measurements derived from the same cross-validation fold are considered paired measurements.
 We used this approach to compare a model trained on true labels with a model trained on shuffled labels (addressing the question, "for the given gene using the given data type, can we predict mutation status better than random"), and to compare a model trained on data type A with a model trained on data type B (addressing the question, "for the given gene, can we make more effective mutation status predictions using data type A or data type B").
 We corrected for multiple tests using a Benjamini-Hochberg false discovery rate correction.
-For all of our experiments, we set a conservative corrected _p_-value threshold of $\alpha = 0.001$; we were able to estimate the number of false positives by examining genes with better performance for shuffled mutation labels than true labels.
+For all of our experiments, we set a conservative corrected threshold of $p = 0.001$; we were able to estimate the number of false positives by examining genes with better performance for shuffled mutation labels than true labels.
 We chose our threshold to ensure that none of these genes were considered significant, since we would never expect permuting labels to improve performance.
 However, our results were not sensitive to the choice of this threshold.
+
+### Multi-omics prediction experiments
+
+To predict mutation presence or absence in cancer genes using multiple data types simultaneously, we concatenated individual datasets into a large feature matrix, then used the same elastic net logistic regression method described previously.
+For this task, we considered only the gene expression, 27K methylation, and 450K methylation datasets.
```

Comment:
Why only these omes and why specifically these omes? It might be nice to add a brief detail to help the reader.